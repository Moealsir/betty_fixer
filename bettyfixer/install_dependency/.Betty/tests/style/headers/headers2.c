typedef unsigned int sample;

struct sample_s
{
	int data;
};

enum sample_e
{
	FIRST = 0,
	SECOND
};

union sample_u
{
	char rgba[4];
	int value;
};

int main(void)
{
	struct dog my_dog;

	my_dog.name = "Django";
	my_dog.age = 3.5;
	my_dog.owner = "Jay";
	printf("My name is %s, and I am %.1f :) - Woof!\n", my_dog.name, my_dog.age);
	return (0);
}
