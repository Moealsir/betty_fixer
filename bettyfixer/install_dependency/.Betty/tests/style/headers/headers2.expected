tests/style/headers/headers2.c:1: WARNING: typedefs should be avoided in .c files
tests/style/headers/headers2.c:3: WARNING: struct definition should be avoided in .c files
tests/style/headers/headers2.c:8: WARNING: enum definition should be avoided in .c files
tests/style/headers/headers2.c:14: WARNING: union definition should be avoided in .c files
total: 0 errors, 4 warnings, 29 lines checked
